kamailio/db_scripts/postgres-specific

```diff
@@ -40,5 +40,6 @@ EOF
 sql_footer() {
 cat << EOF
 ALTER TABLE location_attrs ALTER COLUMN avalue TYPE varchar(512);
+ALTER TABLE presentity ALTER COLUMN etag TYPE varchar(128);
```

Comment:
maybe we should make theses changes  in kamailio ?

Reply:
Hello Daniel @miconda
Could you look this PR.
May be need to extend `etag` column in `presentity` table?
This is because i can see errors as current column size is shor to save `etag` value.
Optionally I can open ticket for this on `kamailio/kamailio` repo.

Reply:
I just updated the schema definition in master so etag size is now 128 for presence tables.